daemon/albatross_console.ml

```diff
@@ -14,9 +14,17 @@ open Lwt.Infix
 
 let pp_unix_error ppf e = Fmt.string ppf (Unix.error_message e)
 
-let active = ref Vmm_core.String_map.empty
+module LMap = Map.Make(Vmm_core.Name.Label)
 
-let read_console id name ring fd =
+(* The console output as a Vmm_ring.t with 1024 entries. The store is a trie,
+   where the key is the path, and the value is pair of "number of unikernels
+   allowed on this path" and "map", which key is the label, and its value a
+   triple: subscribers, active_or_not, ringbuffer. *)
+let state :
+  (int * ((Vmm_commands.version * Lwt_unix.file_descr) list * bool * string Vmm_ring.t) LMap.t) Vmm_trie.t ref =
```

Comment:
Maybe worth it giving part of this type a name? And/or use a record? It's very long :D

Reply:
done in 9b3d2ac